src/DynamoPackages/PackageLoader.cs

```diff
@@ -292,22 +310,25 @@ private void TryLoadPackageIntoLibrary(Package package)
                 PythonServices.PythonEngineManager.Instance.
                     LoadPythonEngine(package.LoadedAssemblies.Select(x => x.Assembly));
             }
-            catch (CustomNodePackageLoadException e)
+            catch (Exception e)
```

Comment:
I think a better coding practice is to catch the most derived exceptions first, then the generic ones later like:
```
catch (CustomNodePackageLoadException e)
{ ... }
catch (DynamoServices.AssemblyBlockedException) 
{ ... }
catch (Exception e)
{ ... }
```

Reply:
Normally yes, that is cleaner.
In this case there code that has to run for all exception types.
This way I avoid code duplication

Reply:
refactored some more and I put back the try catch code as it was